wpimath/src/main/java/edu/wpi/first/math/MatBuilder.java

```diff
@@ -25,7 +24,7 @@ public class MatBuilder<R extends Num, C extends Num> {
    * @param <C> The number of columns of the matrix.
    * @return The constructed matrix.
    */
-  public static final <R extends Num, C extends Num> Matrix<R, C> fill(
+  public static <R extends Num, C extends Num> Matrix<R, C> fill(
```

Comment:
Why remove the `final` marking?

Reply:
Since the class is now marked as final, marking the member function as final is redundant.